currency_rate_date_check/__openerp__.py

```diff
@@ -43,8 +43,8 @@
     """,
     'author': "Akretion,Odoo Community Association (OCA)",
     'website': 'http://www.akretion.com',
-    'depends': ['base'],
-    'data': ['company_view.xml'],
+    'depends': ['account'],
```

Comment:
Use base settings instead in order to avoid change the depends

Reply:
Done.

Regards.